packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.template.ts

```diff
@@ -38,26 +35,43 @@ export function horizontalScrollTemplate<T extends FASTHorizontalScroll>(
                     x => x.view !== "mobile",
                     html<T>`
                         <div
-                            class="scroll scroll-prev"
-                            part="scroll-prev"
+                            class="scroll scroll-previous"
```

Comment:
Do you know if there is a reason we have both classes here (I haven't looked)...we can take as a follow-up.

Reply:
Good catch. I don't think we should have things like this. I've removed that class and updated the test registration. Going forward it will be stronger to be 1:1 with the elements and parts as a more definitive way of providing atyling.